### DISCUSSION TITLE
[Question]: I can't input text into the input field

### DISCUSSION THREAD

**Discussion by filipal** [danny-avila/LibreChat#1912] [Troubleshooting]
### What is your question?

Did I make a mistake during installation or is there another issue? The text input field is grayed out, and I can't type anything or attach a file?

### More Details

C:\LibreChat>docker-compose up
time="2024-02-27T09:15:54+01:00" level=warning msg="The \"UID\" variable is not set. Defaulting to a blank string."
time="2024-02-27T09:15:54+01:00" level=warning msg="The \"GID\" variable is not set. Defaulting to a blank string."
time="2024-02-27T09:15:54+01:00" level=warning msg="The \"UID\" variable is not set. Defaulting to a blank string."
time="2024-02-27T09:15:54+01:00" level=warning msg="The \"GID\" variable is not set. Defaulting to a blank string."
time="2024-02-27T09:15:54+01:00" level=warning msg="The \"UID\" variable is not set. Defaulting to a blank string."
time="2024-02-27T09:15:54+01:00" level=warning msg="The \"GID\" variable is not set. Defaulting to a blank string."

### What is the main subject of your question?

Installation

### Screenshots

<img width="667" alt="2" src="https://github.com/danny-avila/LibreChat/assets/23233693/9c4b4a37-33de-4c01-ba0e-9e6cf242a2a5">
<img width="956" alt="1" src="https://github.com/danny-avila/LibreChat/assets/23233693/92d8cfe5-547c-4526-83f1-16abf89126f5">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
Just set your API key in the menu and you should be good to go
![image](https://github.com/danny-avila/LibreChat/assets/32828263/10e7b330-9a35-4613-b2f2-c388d0158fbd)

**Reply by filipal** [reply]
> fuegovic

Thank you for the quick response. I still have a problem with openAI. I copied my API, but I got an error "Something went wrong. Here's the specific error message we encountered: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors."

**Reply by danny-avila** [reply]
As the docs say, it's a billing issue. You likely need to add a credit card to OpenAI billing.

**Reply by filipal** [reply]
<img width="480" alt="1" src="https://github.com/danny-avila/LibreChat/assets/23233693/c8cee46f-fb4c-4a54-96b7-b6d9e4764ac8">
That part is weird, I have an active subscription plan (plus)

**Reply by fuegovic** [reply]
The API and Plus are different offerings with each their own billing. For the API you need to set it up here: https://platform.openai.com/account/billing/overview

**Reply by filipal** [reply]
Thank you
